Description: Fix FTBFS against readline 6.3
Origin: backport, https://github.com/JuliaLang/julia/commit/d2552e0f03d1dc23d6bb0c89c39da3d73c7aa4e5
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=741824
Last-Update: 2014-03-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ui/repl-readline.c
+++ b/ui/repl-readline.c
@@ -831,7 +831,7 @@ void jl_init_repl(int history)
     rl_instream = fopen("/dev/null","r");
     prompt_length = 7;  // == strlen("julia> ")
     init_history();
-    rl_startup_hook = (Function*)init_rl;
+    rl_startup_hook = (rl_hook_func_t*)init_rl;
 }
 
 static char *prompt_string=NULL;
